app/assets/stylesheets/content/_wiki.md

```diff
@@ -1 +1,126 @@
 # Wiki
+
+Wiki-syntax is used for most textarea-fields within OpenProject. The users have several options to style text.
+
+## Container
+
+```
+<div class="wiki">
+    
+</div>
+```
+
+## Paragraph
+
+```
+<div class="wiki">
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit sed cum quam obcaecati eius nisi tenetur tempora odio minus nulla rerum hic, itaque nam dolorum vel fuga quibusdam, praesentium unde!</p>
```

Comment:
I think in order to judge the layout of a paragraph, it is important to have more than one because especially the spacing between the two is important.